core/relayMsgs.go

```diff
@@ -16,6 +16,9 @@ type RelayMsgs struct {
 
 	Last      bool `json:"last"`
 	Succeeded bool `json:"success"`
+
+	SentSrcMsgIDs []MsgID `json:"sent_src_msg_ids"`
+	SentDstMsgIDs []MsgID `json:"sent_dst_msg_ids"`
```

Comment:
`Sent` is redundant.

Reply:
https://github.com/hyperledger-labs/yui-relayer/commit/286bfe5359804373830a090058994c3dbbb0447b
Change to src_msg_ids, dst_msg_ids